### DISCUSSION TITLE
Fetch-API library for Go / other languages

### DISCUSSION THREAD

**Discussion by goenning** [planetscale/discussion#403] [Q&A] [answered]
Is there a package or example of using the Fetch-API with Go or in other languages? 

Similar to this one https://github.com/planetscale/database-js but not in JavaScript.

**Comment by jgreet** [comment, marked as answer]
There are no other drivers at the moment. PlanetScale may publish another library, but ultimately the plan is to document those endpoints and help people develop their own drivers for additional languages and frameworks.

**Comment by mattrobenolt** [comment]
I'm personally curious, what would you hope to accomplish by using this from other languages? Are there use cases where the typical MySQL driver in Go wouldn't be viable?
